### PR TITLE
Update changelog.md

### PR DESCRIPTION
**Pull request by sameerag**

### CODE REVIEW

File: lib/msal-core/changelog.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
+# 1.2.0
+
+## Features:
+* Iframes Support added (#939, #975, #1053, #1075)
+* Multiple Instances (#980, #1010)
+* Cache changes - to support iframes, multiple instances and other enhancements (#983, #1042, #1067)
+* Redirect URI supported as a request parameter (#1116)
+* fromCache flag added in response to indicate cache vs nw response for a request (#1114)
```
Comment (pkanher617) [inline lib/msal-core/changelog.md]: small typo -> nw should be new?

File: lib/msal-core/changelog.md [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+* Instance Aware support (#969)
+
+## Bugs
+* app set correlation-id is now passed to the service (#1083)
```
Comment (jasonnutter) [inline lib/msal-core/changelog.md]: Can we be specific that this is referring to `request.correlationId`?

File: lib/msal-core/changelog.md [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+* response_mode explicitly set to fragment for all usecases (#1090)
```
Comment (jasonnutter) [inline lib/msal-core/changelog.md]: `usecases` two words?

File: lib/msal-core/changelog.md [hunk 4]
```diff
@@ -1,3 +1,35 @@
...
+* verbose messages and monitoring for timeout errors (#1098, #1113)
+* handle loginRedirect set to null (#1047)
```
Comment (jasonnutter) [inline lib/msal-core/changelog.md]: Consistent sentence casing

File: lib/msal-core/changelog.md [hunk 5]
```diff
@@ -1,3 +1,35 @@
+# 1.2.0
+
+## Features:
+* Iframes Support added  (#939, #975, #1053, #1075); msal js now added support for authentication in applications embedded in iframes which implies that an application can now call `loginSilent()`, `acquireTokenSilent()` and `acquireTokenPopup()` from iframes.
+* `redirectUri` supported as a request parameter (#1116); This feature in conjunction with iframes support improves performance, by providing the application the capability to avoid a full reload of a SPA on redirect. Please refer to the sample in the release [notes](1.2.0 release notes) for the usage details
```
Comment (jasonnutter) [inline lib/msal-core/changelog.md]: Link at the end isn't valid.

**Comment by DarylThayil** [comment]
lgtm

**Pull request merged**